plugins/woocommerce/includes/admin/meta-boxes/class-wc-meta-box-order-data.php

```diff
@@ -35,8 +35,10 @@ class WC_Meta_Box_Order_Data {
 
 	/**
 	 * Init billing and shipping fields we display + save.
+	 *
+	 * @param WC_Order $order Order object.
 	 */
-	public static function init_address_fields() {
+	public static function init_address_fields( $order ) {
```

Comment:
This is a public API, so let's ensure we don't break it for others who might be relying on it. To do so, please make sure it works with no order passed in.

Reply:
Would something like this be acceptable?

`public static function init_address_fields( $order = null ) {`

and then

`'options' => array( '' => __( 'Select a country / region&hellip;', 'woocommerce' ) ) + ( is_a( $order, 'WC_Order' ) ?  $order->get_allowed_billing_countries_for_edit() : WC()->countries->get_allowed_countries() ),`